WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -345,7 +345,7 @@ protected function check_argument_tokens( $context ) {
 
 		if ( empty( $tokens ) || 0 === count( $tokens ) ) {
 			$code = $this->string_to_errorcode( 'MissingArg' . ucfirst( $arg_name ) );
-			if ( 'domain' !== $arg_name || ! empty( $this->text_domain ) ) {
+			if ( ( 'domain' !== $arg_name || ! empty( $this->text_domain ) ) && ! in_array( 'default', $this->text_domain, true ) ) {
```

Comment:
The braces are incorrectly placed on this line which is one of the reasons why the unit tests are failing.
```php
if ( 'domain' !== $arg_name || ( ! empty( $this->text_domain ) && ! in_array( 'default', $this->text_domain, true ) ) ) {}
```